test/showcase/schema.js

```diff
@@ -59,13 +59,14 @@ export default {
       },
     },
   },
-  country: {
+  package: {
     type: String,
     required: true,
     length: {
       min: 2,
-      max: 2,
+      max: 200,
     },
```

Comment:
The length is not relevant to validate anymore if we have a dedicated validator :)
```suggestion
```